Dockerfile

```diff
@@ -1,11 +1,10 @@
 FROM golang:1.14.1 as builder
-WORKDIR /go/src/vvgo
+WORKDIR /go/src/app
 COPY . .
 RUN go mod download
-RUN CGO_ENABLED=0 GOOS=linux go build -v -o vvgo
```

Comment:
We can add the test commands here during build phase.

Reply:
There's a dedicated section of the GH Action workflow for testing, that I commented out for now. Once this lands, you can add the testing stuff there.